### PR TITLE
make opening times a combo of a page and opening hours

### PR DESCRIPTION
**Pull request by jamesgorrie**
References #2490 

## Who is this for?
🚁 

## What is it doing for them?
Make opening hours a combo-like page, mixing a query of opening hours and the opening hours page.

I don't like it, but it works, my pragmatism is stretching.

This allows the page to be included in content lists and search results.

Another way would be to make this a specific, singleton type in prismic, and map all the information in that type - then we could just query it.

This might be useful if we wanted to have a special promo for this.

### CODE REVIEW

**Comment by jamesgorrie** [comment]
I've also added a perf page to our datastudio doc.

File: common/views/components/OpeningHours/OpeningHours.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ class OpeningHours extends Component<Props, State> {
       <Fragment>
         {groupedVenues && !groupedVenues[Object.keys(groupedVenues)[0]].hours &&
           <p className={spacing({s: 2}, {margin: ['bottom']})}>
-            <a className={font({s: 'HNL6', m: 'HNL5'})} href='https://wellcomecollection.org/info/opening-times'>
+            <a className={font({s: 'HNL6', m: 'HNL5'})} href='https://wellcomecollection.or/opening-times'>
```
Comment (gestchild) [inline common/views/components/OpeningHours/OpeningHours.js]: typo .or/

Comment (gestchild) [reply common/views/components/OpeningHours/OpeningHours.js]: and will need to add ^/opening-times to router

Comment (jamesgorrie) [reply common/views/components/OpeningHours/OpeningHours.js]: @gestchild Ah yes - fixed.

**Pull request merged**